src/ontology/cl-edit.owl

```diff
@@ -27884,6 +27925,45 @@ AnnotationAssertion(rdfs:label obo:CL_4023053 "spinal interneuron syanpsing Betz
 SubClassOf(obo:CL_4023053 obo:CL_0008049)
 SubClassOf(obo:CL_4023053 ObjectSomeValuesFrom(obo:RO_0002120 obo:CL_0005000))
 
+
+# Class: obo:CL_4023054 (mesothelial fibroblast)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:26106328") obo:IAO_0000115 obo:CL_4023054 "A mesothelial cell that has undergone mesothelial-to-mesenchymal transition (MMT) to become a fibroblast cell.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023054 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023054 "mesothelial fibroblast")
+SubClassOf(obo:CL_4023054 obo:CL_0000057)
+SubClassOf(obo:CL_4023054 obo:CL_0000077)
+
+# Class: obo:CL_4023056 (vascular leptomeningeal cell (Mus musculus))
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30096314"^^xsd:string) obo:IAO_0000115 obo:CL_4023056 "A type of mouse mesothelial fibroblast that is derived from the neural crest and is a key component of the pia and arachnoid membranes surrounding the brain.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023056 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023056 "VLMC (Mus)")
+AnnotationAssertion(oboInOwl:inSubset obo:CL_4023056 <http://purl.obolibrary.org/obo/cl#BDS_subset>)
+AnnotationAssertion(rdfs:label obo:CL_4023056 "vascular leptomeningeal cell (Mus musculus)")
+SubClassOf(obo:CL_4023056 obo:CL_4023051)
+SubClassOf(obo:CL_4023056 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_10090))
+SubClassOf(Annotation(oboInOwl:hasDbXref "PMID:30096314"^^xsd:string) obo:CL_4023056 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P11087))
+SubClassOf(Annotation(oboInOwl:hasDbXref "PMID:30096314"^^xsd:string) obo:CL_4023056 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P26618))
+SubClassOf(Annotation(oboInOwl:hasDbXref "PMID:30096314"^^xsd:string) obo:CL_4023056 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P51885))
+
+# Class: obo:CL_4023057 (arachnoid barrier cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:23298861") obo:IAO_0000115 obo:CL_4023057 "a cell that makes up the tight-junctioned layer of the leptomeninges that functions as the physiologic barrier between the CSF in the subarachnoid space and the fenestrated capillaries in the dura.")
```

Comment:
Make part of arachnoid mater?  Also simplify def and make sure it doesn't duplicate def of tissue it is part of.  TO push this through, split out ABC to separate PR.